vectordraw/templates/html/vectordraw_edit.html

```diff
@@ -0,0 +1,372 @@
+{% load i18n %}
+<div class="editor-with-buttons">
+  <div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
+    <ul class="list-input settings-list">
+      {% for field in fields %}
+        <li class="field comp-setting-entry metadata_entry {% if field.is_set %}is-set{% endif %}"
+            data-field-name="{{field.name}}"
+            data-default="{% if field.type == 'boolean' %}{{ field.default|yesno:'1,0' }}{% else %}{{ field.default|default_if_none:"" }}{% endif %}"
+            data-cast="{{field.type}}">
+          <div class="wrapper-comp-setting{% if field.type == "set" %} metadata-list-enum {%endif%}">
+            <label class="label setting-label" for="xb-field-edit-{{field.name}}">{{field.display_name}}</label>
+
+            {% if field.type == "boolean" %}
+              <select class="field-data-control"
+                      id="xb-field-edit-{{field.name}}"
+                      aria-describedby="{{field.name}}-help">
+                <option value="1" {% if field.value %}selected{% endif %}>
+                  True {% if field.default %}&nbsp;&nbsp;&nbsp;&nbsp;(Default){% endif %}
+                </option>
+                <option value="0" {% if not field.value %}selected{% endif %}>
+                  False {% if not field.default %}&nbsp;&nbsp;&nbsp;&nbsp;(Default){% endif %}
+                </option>
+              </select>
+            {% elif field.type == "string" %}
+              <input type="text"
+                     class="field-data-control"
+                     id="xb-field-edit-{{field.name}}"
+                     value="{{field.value|default_if_none:""}}"
+                     aria-describedby="{{field.name}}-help">
+            {% elif field.type == "integer" or field.type == "float" %}
+              <input type="number"
+                     class="field-data-control"
+                     id="xb-field-edit-{{field.name}}"
+                     {% if field.step %} step="{{field.step}}" {% elif field.type == "integer" %} step=1 {% endif %}
+                     {% if field.max %} max="{{field.max}}" {% endif %}
+                     {% if field.min %} min="{{field.min}}" {% endif %}
+                     value="{{field.value|default_if_none:""}}"
+                     aria-describedby="{{field.name}}-help">
+            {% elif field.type == "text" or field.type == "html" %}
+              <textarea class="field-data-control"
+                        data-field-name="{{field.name}}"
+                        id="xb-field-edit-{{field.name}}"
+                        aria-describedby="{{field.name}}-help"
+                        rows=10 cols=70>{{field.value}}</textarea>
+            {% else %}
+              Unsupported field type. This setting cannot be edited.
```

Comment:
Everything else is nicely marked with `{% trans %}`, you might as well get this one too.